gio/vendor.md

```diff
@@ -1178,11 +1178,11 @@ application is not expected to manually register itself and check
 if it is the primary instance. Instead, the `main` function of a
 `Application` should do very little more than instantiating the
 application instance, possibly connecting signal handlers, then
-calling `ApplicationExt::run`. All checks for uniqueness are done
+calling `Application::run`. All checks for uniqueness are done
```

Comment:
This happened because `Application::run` now ignored and actually `ApplicationExtManual::run`.